modules/auxiliary/scanner/rdp/cve_2019_0708_bluekeep.rb

```diff
@@ -6,6 +6,7 @@
 class MetasploitModule < Msf::Auxiliary
   require 'rc4'
   include Msf::Exploit::Remote::Tcp
```

Comment:
Ideally, `Msf::Exploit::Remote::RDP` should include `Msf::Exploit::Remote::Tcp` if it uses it.

Reply:
Thanks, moved this and the require for `rc4` over to the library where they should have been.